modules/apps/permission-template/permission-template-api/src/main/resources/com/liferay/permission/template/exception/packageinfo

```diff
@@ -0,0 +1 @@
+version 1.0.0
```

Comment:
@ces-quanhuynh, I think the change in `packageinfo` files should be committed under "create module" commit